<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HashTable Test</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTextEdit" name="mainTextArea"/>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="lvl1Layout">
       <item>
        <layout class="QVBoxLayout" name="upLvl2Layout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Additional sheet</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="additionalTextArea"/>
         </item>
         <item>
          <widget class="QLineEdit" name="searchEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="searchButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Search</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="testButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="text">
            <string>Test</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="downLvl2Layout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_1">
           <item>
            <widget class="QLabel" name="countToGenerateLabel">
             <property name="text">
              <string>Elements generate</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="numberGenerateEdit">
             <property name="text">
              <string>500000</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="lvl4Layout">
             <item>
              <widget class="QPushButton" name="generateButton">
               <property name="text">
                <string>Generate</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="stateButton">
               <property name="text">
                <string>State</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="clearButton">
               <property name="text">
                <string>Clear</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
